packages/app-project/src/screens/ProjectAboutPage/ProjectAboutPage.js

```diff
@@ -0,0 +1,153 @@
+import { Box, Grid, Heading, Text } from 'grommet'
+import { arrayOf, bool, object, shape, string } from 'prop-types'
+import styled, { css, withTheme } from 'styled-components'
+import { withResponsiveContext } from '@zooniverse/react-components'
+
+/** Components */
+import Announcements from '@components/Announcements'
+import ProjectHeader from '@components/ProjectHeader'
+import ZooHeaderWrapper from '@components/ZooHeaderWrapper'
+import { ZooFooter, SpacedHeading } from '@zooniverse/react-components'
+import AboutSidebar from './components/AboutSidebar'
+import AboutDropdownNav from './components/AboutDropdownNav'
+import TeamMember from './components/TeamMember'
+import AboutMarkdownz from './components/AboutMarkdownz/AboutMarkdownz'
+
+const FullHeightBox = styled(Box)`
+  min-height: 98vh;
+  border-width: 0 1px;
+  border-style: solid;
+  ${props =>
+    css`
+      border-color: ${props.theme.global.colors['light-3']};
+    `}
+  ${props =>
+    css`
+      background: ${props.theme.global.colors['white']};
+    `}
+`
+
+const PageHeading = styled(Heading)`
+  font-weight: normal;
+  font-size: 40px;
+  margin-bottom: 30px;
+`
+
+const SidebarHeading = styled(SpacedHeading)`
+  padding: 5px 20px;
+`
+
+const TeamHeading = styled(Text)`
+  letter-spacing: 0.067rem; // same as SpacedText component
+  text-transform: uppercase;
+  color: black;
+  font-weight: bold;
+  margin-bottom: 14px;
+`
+
+const StyledList = styled(Box)`
+  padding: 0;
+`
+
+function ProjectAboutPage({
+  aboutPageData,
+  inBeta,
+  projectDisplayName,
+  teamArray,
+  screenSize
+}) {
+  const { content = '', title = '' } = aboutPageData
+
+  const isTeamPage = title.toLowerCase().includes('team')
+
+  return (
+    <Box>
+      <ZooHeaderWrapper />
+      <ProjectHeader />
+      <Announcements />
```

Comment:
These look great.

There's a `StandardLayout` component which will take care of the headers, announcements and footer for you.

```jsx
import StandardLayout from '@shared/components/StandardLayout'
…
<StandardLayout>
//… page content components
</StandardLayout>
```